### ISSUE TITLE
ifconfig sample creates a poor impression

### ISSUE THREAD

**Issue by jhoneill** [PowerShell/Crescendo#152]
- It has place holders where help should be, 
- It doesn't parse output 
- It simply just turns ifconfig into invoke-ifconfig. 

Why would anyone use it ? 
Several of the examples appear to turn `command` into `invoke-command with longer parameter names`  (dd, ls, tar, who) without parsing the output.

**Comment by theJasonHelmick** [comment]
Thanks for the feedback @jhoneill.  The samples are not intended to be complete solutions, rather samples of a particular feature or task.  Having said that, I do agree that they can be improved, including some of the points you mention, to better help folks working with Crescendo.

**Comment by jhoneill** [comment]
@theJasonHelmick  Yes, understood, but samples shipped with code should be really knock-your-socks-off examples; some of those included are good, and this one, TBH, just isn't.

**Comment by JamesWTruher** [comment]
I don't agree that all the samples should be knock-your-socks-off examples. _very_ simple examples should also be present to show a complete story.

**Comment by jhoneill** [comment]
@JamesWTruher  

YES, simple is good if it tells a story

Functionally this sample can be replaced with    `new-alias -Name Invoke-IfConfig -Value ifconfig` 

The json contains    
`   "Description": "This is the description for a parameter", `    
which just looks like someone isn't trying.

**Comment by JamesWTruher** [comment]
I'll update the description.
It doesn't matter if it can be replaced by an alias, that is _not_ the point. It shows that a very simple example is possible.

**Comment by theJasonHelmick** [comment]
Thanks James (@jhoneill ). We are all trying.  as mentioned, I'll go through and take a look at places we could add additional examples and improvements in a future milestone.
